en/sensor/tachometers.md

```diff
@@ -1,6 +1,6 @@
 # Tachometers (Revolution Counters)
 
-Tachometers (also known as [revolution-counter sensors](https://en.wikipedia.org/wiki/Tachometer#In_automobiles,_trucks,_tractors_and_aircraft)) are *highly recommended* for rotor-wing frames because they allow the autopilot to detect stall or another rotor failure (for rotor-flight it is the rotation of blades that guarantees lift not airspeed or ground speed).
+Tachometers (also known as [revolution-counter sensors](https://en.wikipedia.org/wiki/Tachometer#In_automobiles,_trucks,_tractors_and_aircraft)) can be used to measure the rate of rotation turning vehicle parts like rotors or wheels. Example applications of tachometers are the measuring of the rate of rotation of autogyro rotors or gasoline engines.
```

Comment:
So the point here is that in theory they can be useful but PX4 does not use them "specifically"? Should we make that clear?

Reply:
Yes, I think right now it only logs the data in a generic rpm message. But that could be used in the future or custom implementation.